SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UnresolvedReferenceGymTest/UnresolvedReferenceGymTest.cpp

```diff
@@ -0,0 +1,57 @@
+
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "UnresolvedReferenceGymTest.h"
+
+AUnresolvedReferenceGymTest::AUnresolvedReferenceGymTest()
+	: Super()
+{
+	Author = "Ajanth";
+	Description = TEXT("Tests what happens when structs with references to actors whose entity have not been created yet are replicated.");
```

Comment:
Perhaps we should include what the success state is of the test is in the description.

Something like: "Test whether unresolved actor references (actors whose entities have not yet been created) inside a replicated struct are eventual resolve"